### DISCUSSION TITLE
Request integration support for tenderly supporting Darwinia

### DISCUSSION THREAD

**Discussion by hackfisher** [ringdao-community-guild/discussion#113] [Integration]
Crab is optional and good to have.

https://docs.tenderly.co/supported-networks-and-languages

Tally and DSafe simulation need this.

**Comment by nblogist** [comment]
Re-pinged tenderly for a response.

**Reply by nblogist** [reply]
@hackfisher who else should be on the call with them? I just got back the request to book a call! We can have a call next week!

**Reply by hackfisher** [reply]
echo.hu@itering.io
guantong.ji@itering.io
aki.wu@itering.io

**Comment by nblogist** [comment]
Just had the call with them - @wuminzhe was there too!
So Nevena will share the pricing but overall - things look very good, they even include the testnet integrations as well.

These are my meeting notes:
```
Node Support Pricing -  You get tooling and the node - integration is full - you get all services not only the simulation - the whole platform and even the RPC node this is all included in the pricing

Integrates Mainnet and test net- one price for both - on first 2 tiers the testnet isn't available to others publicly

tier 3+ will get you on the website of tenderly - increases discoverability
```

I will ping you when i get the documents, making a TG group now!

**Reply by nblogist** [reply]
TG group is made and pricing offer is shared there!
